lms/djangoapps/certificates/apis/v0/permissions.py

```diff
@@ -0,0 +1,30 @@
+"""
+This module provides a custom DRF Permission class for supporting the course certificates
+to Admin users and users whom they belongs to.
+"""
+
+from django.conf import settings
+from django.contrib.auth.models import User
+from rest_framework.permissions import IsAdminUser
+
+from openedx.core.djangoapps.user_api.accounts.serializers import get_profile_visibility
+
+
+class IsOwnerOrPublicProfile(IsAdminUser):
+    """
+    Method that will ensure whether the requesting user is staff or
+    the user whom the certificate belongs to
+    """
+    def has_permission(self, request, view):
+        requested_profile_username = view.kwargs.get('username')
+        is_staff = super(IsOwnerOrPublicProfile, self).has_permission(request, view)
```

Comment:
Why we are still checking for staff here while we already have 'IsAdminUser` for that purpose?